master/internal/db/postgres_users.go

```diff
@@ -2,15 +2,19 @@ package db
 
 import (
 	"crypto/ed25519"
+	"crypto/rsa"
+	"encoding/json"
 	"fmt"
 	"strings"
 	"time"
 
+	"github.com/golang-jwt/jwt"
```

Comment:
since I had to double check this was the right library, could you add `github.com/dgrijalva/jwt-go` to the golangci-lint blacklist?